src/PowerupSubsystem.cpp

```diff
@@ -12,7 +12,9 @@ namespace {
     const float POWERUP_DISTANCE_THRESHOLD = 2.5f;
     const int POWERUP_LOCK_FRAMES = 30;
     const glm::vec3 POWERUP_LOCATION_LIMITS = glm::vec3(10.0f, 1.5f, 10.0f);
-
+    glm::vec3 charcoal_locations[20];
+    int mound_index = 0;
+    int power_loc = 1;
```

Comment:
These 3 variables would be better as private members since they are inherently part of the power up subsystem, just makes more sense.